free-1.md

```diff
@@ -1,54 +1,76 @@
 # Free project 1
 
 ## The user and a language
-This section describes who the project would serve and why a language might be a
-good way to meet their needs.
+The first language I would like to consider is one for music composers. It would help them organize ideas about compositions into structures they create, simulating a top-down approach to composing music.
 
 
 ### What's the need?
 _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+My idea satisfies the need of organization when composing music. We will be helping composers of any skill level organize their compositions. The main reason why I'd like to build a language for organizing composition structures is because a current person's experiences in composing is done on paper or in a language not built for organizing composition structures. There are a lot (like, really, a lot) of langauges that focus on creating music and aiding musicians. But I haven't come across one solely dedicated to organization yet. I hope that the language I am thinking of helps people efficiently organize their thoughts in compositions and frees them to think of other aspects of composing music such as harmony, rhythm, and textures. From here, the hope is that the less time spent thinking about organization leads to more time/efficiency on the other aspects.
```

Comment:
What do you mean by organizing composition structures? I don't really follow what a composition structure is, and how/why you would organize it.